addons/transformations.md

```diff
@@ -108,17 +109,104 @@ Note the overall syntax is the same.
 
 ::: tab JRuby
 
-The script-prefix is `rb`.
+The script file name is `stringlength.rb` and the transformation is `RB(stringlength.rb)`.
 
 ```ruby
 "String has #{input.length} characters"
 ```
 
 :::
 
+::: tab Groovy
+
+The script file name is `stringlength.groovy` and the transformation is `GROOVY(stringlength.groovy)`.
+
+```groovy
+"String has ${input.length()} characters"
+```
+
+:::
+
 ::::
 
-Currently the `SCRIPT` transformation is not available as profile.
+### Inline Script Transformation
+
+A simple transformation rule can also be given as an inline script. The inline script should
+start with the `|` character. Quotes within the script may need to be escaped with a `\` when
+used within another quoted string such as in sitemaps or text configurations.
+
```

Comment:
^ see above. Put each sentences into separate line.

Reply:
Should I put "The inline script should..." also in its own separate line?

Reply:
Yes, correct: each sentence should start at a new line (the rendering will concatenate them into a one-liner).
FYI, to be honest, I do not know if this is required in the case when you have multiple sentences in table columns nor do I even know if that would work. For that particular case I got in touch with @Confectrician to ask him. As a result you don't have to do that in table columns.

Reply:
OK I've split them up.

Reply:
You can't break them into multiple lines inside a table. All the columns would have to all be in one single line per row.